src/OmenThumbnailMapping.sol

```diff
@@ -5,24 +5,31 @@ import {IERC20} from "../lib/openzeppelin-contracts/contracts/interfaces/IERC20.
 
 /// @dev Mapping from market's address to image's IPFS hash
 contract OmenThumbnailMapping {
+    event ImageUpdated(address indexed marketAddress, bytes32 image_hash, address indexed changer);
+
     mapping(address => bytes32) private marketAddressToIPFSHash;
     mapping(address => address) private marketAddressToLatestImageChanger;
 
     /// @dev Get IPFS hash of thumbnail for the given market.
-    function get(address marketAddress) external view returns (bytes32) {
+    function get(address marketAddress) public view returns (bytes32) {
```

Comment:
Changed all from external to public because now one of these functions is called by another one, so to keep it consistent.